be/src/exec/analytic_eval_node.cpp

```diff
@@ -710,7 +709,13 @@ Status AnalyticEvalNode::process_child_batch(RuntimeState* state) {
 
     // Transfer resources to _prev_tuple_pool when enough resources have accumulated
     // and the _prev_tuple_pool has already been transfered to an output batch.
-    if (_curr_tuple_pool->total_allocated_bytes() > MAX_TUPLE_POOL_SIZE &&
+
+    // The memory limit of _curr_tuple_pool is set by the fixed value MAX_TUPLE_POOL_SIZE
```

Comment:
there is no MAX_TUPLE_POOL_SIZE， you deleted it before

Reply:
ok，i will change the comment